FEATURES.md

```diff
@@ -0,0 +1,17 @@
+# Proposing Features
+
+A feature request or proposal is an issue opened on a repo in opensearch-project that states an end-user problem, and proposes a solution in the form of a feature. It should meet the following criteria.
+
+- Is a feature.
+- Fits in the scope of the repo and project.
```

Comment:
The word scope there is doing a lot of heavy lifting.  Also, I suspect for some features we might not know if it should be in a plug in or core, so I'm not sure if this question is going to tell us anything .

Reply:
I couldn't come up with better. I could say "related", but I think "scope" is better.

Reply:
Hmmm....
"It is a good fit with existing features and the overall direction of the repo and project"
"It is appropriate for the repo and the project"
"It is directional for the repo and project"
"It aligns with the architectural direction of the repo and project"

If you don't like any of those, we can stick with scope.

Reply:
I like the second one, added "Is a good fit with existing features and the overall direction of the repo and project."